cmd/manager/backup/compressor/main.go

```diff
@@ -0,0 +1,64 @@
+//
+// Copyright (C) 2019 Vdaas.org Vald team ( kpango, kou-m, rinx )
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// Package main provides program main
+package main
+
+import (
+	"context"
+
+	"github.com/vdaas/vald/internal/log"
+	"github.com/vdaas/vald/internal/runner"
+	"github.com/vdaas/vald/internal/safety"
+	"github.com/vdaas/vald/pkg/manager/backup/compressor/config"
+	"github.com/vdaas/vald/pkg/manager/backup/compressor/usecase"
+)
+
+const (
+	// version represent the version
+	version    = "v0.0.1"
+	maxVersion = "v0.0.10"
+	minVersion = "v0.0.0"
+)
+
+func main() {
```

Comment:
![badge](https://img.shields.io/badge/review-must-red.svg)
Rebase master and wrap `runner.Do` block with `RecoverFunc` function.